packages/bundler/src/interfaces/IBundler.ts

```diff
@@ -7,4 +7,5 @@ export interface IBundler {
         chainId: ChainId
     ): Promise<UserOpGasFieldsResponse>
     sendUserOp(userOp: UserOperation, chainId: ChainId): Promise<UserOpResponse>
+    getUserOpReceipt(userOpHash: string, chainId: ChainId): Promise<UserOpReceipt>
```

Comment:
One more method is missing, getUserOpByHash as per ERC specifications.

Reply:
added